Sprint-1/implement/max.js

```diff
@@ -1,4 +1,15 @@
 function findMax(elements) {
+  if (elements.length === 0) return -Infinity;
+
+  let max = -Infinity;
+
+  for (let i = 0; i < elements.length; i++) {
+    if (typeof elements[i] === "number") {
+      max = Math.max(max, elements[i]);
+    }
+  }
```

Comment:
What do you expect from the following function calls (on extreme cases)?
Does your function returns the value you expected?

```
findMax([-Infinity])
findMax([-Infinity, 0])
findMax([NaN])
findMax([0, NaN, 1])
```

Reply:
thank you for the review.
findMax([-Infinity]) : My expected Output is -Infinity. The loop iterates over the array, and since -Infinity is a number, Math.max(max, -Infinity) is calculated. Initially, max = -Infinity, so it remains -Infinity.
findMax([-Infinity, 0]) : Expected Output: 0.
findMax([NaN]) : Expected Output: -Infinity.
findMax([0, NaN, 1]) : Expected Output: 1.